openbb_platform/CONTRIBUTING.md

```diff
@@ -451,66 +533,38 @@ Please refer to [OpenBBTerminal docs](https://docs.openbb.co/terminal/installati
     pip install poetry
     ```
 
-4. Install the `openbb-core`:
-
-    ```bash
-    cd OpenBBTerminal/openbb_platform/platform/core/
-    poetry install
-    ```
-
-5. Install dependencies:
+4. Install the packages using the `dev_install.py` script located in the `openbb_platform` folder:
 
     ```bash
-    cd OpenBBTerminal/openbb_platform/extensions/stocks/ # or any other extension
-    poetry install
+    python dev_install.py
     ```
 
-> When installing the dependencies using poetry we ensure that dependencies are being installed in editable mode, which is the most straightforward way to develop on top of the Platform.
-
-<details>
-<summary>Install all dependencies in editable mode at once</summary>
+   > To install all the packages, including extras, use the `-e` argument with the above script.
 
-For development purposes, one can install every available extension by running a custom shell script:
+## How to create a PR?
 
-Navigate to `/OpenBBTerminal/openbb_platform`
+To create a PR to the OpenBB Platform, you'll need to fork the repository and create a new branch.
 
-Run `python dev_install.py`.
-</details>
+1. Create your Feature Branch, e.g. `git checkout -b feature/AmazingFeature`
+2. Check the files you have touched using `git status`
+3. Stage the files you want to commit, e.g.
+   `git add openbb_terminal/stocks/stocks_controller.py openbb_terminal/stocks/stocks_helper.py`.
+   Note: **DON'T** add any files with personal information.
+4. Write a concise commit message under 50 characters, e.g. `git commit -m "meaningful commit message"`. If your PR
+   solves an issue raised by a user, you may specify such issue by adding #ISSUE_NUMBER to the commit message, so that
+   these get linked. Note: If you installed pre-commit hooks and one of the formatters re-formats your code, you'll need
+   to go back to step 3 to add these.
 
-> In order to install any other custom extension or provider, you'd follow the exact same steps as above.
+### Install pre-commit hooks
 
-## Python package
+To install pre-commit hooks, run `pre-commit install` in the root of the repository.
 
-### Overview
+### Branch Naming Conventions
 
-One of the interfaces we provide to users is the `openbb` python package.
+The accepted branch naming conventions are:
 
-The code you will find in this package is generated from a script and it is just a
-wrapper around the `openbb-core` and any installed extensions.
+- `feature/feature-name`
+- `hotfix/hotfix-name`
+- `release/2.1.0` or `release/2.1.0rc0`.
```

Comment:
Do we want people making PRs to the `/release` branch?

Reply:
Don't think so. I copied this part from the v3 CONTRIBUTING.md docs. We can remove it.

Reply:
It's just helpful info for what we consider good naming conventions for PRs.